src/user-utilities.js

```diff
@@ -50,7 +50,21 @@ function createUser(firstName, lastName, email, phoneNumber) {
 
 }
 
+function   manageUsers(user, a) {
```

Comment:
perhaps a better name for your second arguement would be action, if you could clarify what it means? @jvannier

Reply:
Should we manage users in one function or have separate functions?